### DISCUSSION TITLE
Neo 1 + rizer

### DISCUSSION THREAD

**Discussion by benjaminfrombe** [cagnulein/qdomyos-zwift#658] [Q&A]
Hi,

I purchased the IOS app yesterday since I recently bought an elite rizer and noticed a substantial delay when connecting the rizer to my NEO. I was hoping to be able to qz as a bridge so the rizer would work without the delay. But I can't really manage to get it working. How should I configure the NEO ? I tried with linking it as a power meter, and switch "power as a treadmill" on, with this setting I am able to select my phone as a device, but no data comes through ...

I noticed only NEO2 is in the compatibility list, but from what I hear the NEO 1 isn't all that different so I was hoping it would work ...

any ideas ? 

Thank you, 
Benjamin

**Comment by cagnulein** [comment]
Hi @benjaminfrombe ! Ok let's start from the NEO2. Create a ticket about it adding a nrfconnect screenshot about it. I will add it. After that, we can link the rizer directly to QZ.
Let me know

**Comment by benjaminfrombe** [comment]
ok, created an issue and added NRFconnect screenshots and log. Thanks a lot !

Benjamin

**Comment by cagnulein** [comment]
#659
